tflite2onnx/op/squareddifference.py

```diff
@@ -0,0 +1,118 @@
+import logging
```

Comment:
Suggest naming this file with a `_`, i.e. `squared_difference.py`.

Reply:
https://github.com/jackwish/tflite2onnx/pull/48/files#diff-1e482a50b4377b2620d1340cb86b9dbec61b7ccb8a7e7620955a16451624d943R1